exercises/concept/booking-up-for-beauty/.docs/instructions.md

```diff
@@ -40,11 +40,13 @@ Description("7/25/2019 13:45:00")
 // Output: "You have an appointment on Thursday, July 25, 2019, at 13:45."
 ```
 
-## 5. Return the anniversary date
+## 5. Return the anniversary date of the salon's opening
```

Comment:
Can you please apply the headline change also in the hints.md file? They need to match.

Reply:
Done.